spokestack/asr/google/speech_recognizer.py

```diff
@@ -0,0 +1,104 @@
+"""
+This module contains the google asr speech recognizer
+"""
+import logging
+from queue import Queue
+from threading import Thread
+from typing import Any
+
+import numpy as np  # type: ignore
+from google.cloud import speech  # type: ignore
+from google.oauth2 import service_account  # type: ignore
+
+from spokestack.context import SpeechContext
+
+
+_LOG = logging.getLogger(__name__)
+
+
+class GoogleSpeechRecognizer:
+    """ Transforms speech into text using Google's ASR.
+
+     Args:
+         cred_path (str): path to google credentials
+         language (str): language of input audio
+         sample_rate (int): sample rate of the input audio (Hz)
+         **kwargs (optional): additional keyword arguments
+     """
+
+    def __init__(
+        self, cred_path, language="en-US", sample_rate=16000, **kwargs,
+    ) -> None:
+
+        credentials = service_account.Credentials.from_service_account_file(cred_path)
+        self._client = speech.SpeechClient(credentials=credentials, **kwargs)
```

Comment:
we don't want to forward spokestack kwargs to the speech client, because we will be sending it things like vad configuration, etc.
I'm guessing this is just working by accident right now

Reply:
I thought the point of kwargs was that you could add arbitrary keyword arguments as long as each class has `**kwargs` in its signature.

Reply:
right, but we don't want to make assumptions about the speech client's parameters or accidentally pass a conflicting parameter to them